src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { ChangeEvent, useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { normalizeMovieData } from '../../utils/normalizeMovieData';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [hasError, setErrorStatus] = useState(false);
+  const [isLoading, setLoadingStatus] = useState(false);
+
+  const handleQuery = (event: ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    setErrorStatus(false);
+  };
+
+  const handleMovie = async () => {
+    setLoadingStatus(true);
+
+    try {
+      const result = await getMovie(query);
+
+      if ((result as MovieData).Title) {
+        setMovie(normalizeMovieData(result as MovieData));
```

Comment:
Isn't it more logical to use this typecasting only once to assign it to `result` instead of doing it everytime?